dev-libs/OpenNI/OpenNI-9999.ebuild

```
@@ -52,7 +60,8 @@ src_prepare() {
 		echo "" > ${i}
 	done
 
-	find . -type f -print0 | xargs -0 sed -i "s:\".*/SamplesConfig.xml:\"${EPREFIX}/usr/share/${PN}/SamplesConfig.xml:" || die
+	find . -type f -print0 |
+		xargs -0 sed -i "s:\".*/SamplesConfig.xml:\"${EPREFIX}/usr/share/${PN}/SamplesConfig.xml:" || die
```

Even though one could argue checking the exit status of `find` may not be necessary, I'd simply always use pipestatus (from eapi9-pipestatus.eclass) with pipes:

```suggestion
local status
find . -type f -print0 |
		xargs -0 sed -i "s:\".*/SamplesConfig.xml:\"${EPREFIX}/usr/share/${PN}/SamplesConfig.xml:"
 status=$(pipestatus -v)  || die "faile dto sed SamplesConfig, (PIPESTATUA: ${status})"
```

---

will check about `eapi9-pipestatus.eclass` later.